crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -96,7 +96,8 @@ impl CallContractCommand {
 			full_message.push_str(&format!(" --message {}", message));
 		}
 		if !self.args.is_empty() {
-			full_message.push_str(&format!(" --args {}", self.args.join(",")));
+			let args: Vec<_> = self.args.iter().map(|a| format!("\"{a}\"")).collect();
```

Comment:
Args are always quoted for safety - complex types might have spaces or commas in their value entry and its best to wrap with "".

Reply:
Nice